### PR TITLE
Add landing page

### PR DESCRIPTION
**Pull request by nelsonmestevao**
[Figma](https://www.figma.com/file/vrq1JmgubKLKgsLQf1D6Bl/Landing-Page-CoderDojo-Braga?node-id=0%3A1)

### CODE REVIEW

**Comment by nelsonmestevao** [comment]
#### 📝Changed routes:


| [**/**](https://shuriken-mpevm5eip-cesiuminho.vercel.app/) |
|:-:|
|<a href="https://shuriken-mpevm5eip-cesiuminho.vercel.app/">Screenshot of /</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/PWa_QQQPbJEIhNNS8DjaYYd_ugHUjJBDgcbwwlmudSs-DdfubvQ2xAxyxBFNd5tf-mJ1H9lcUlYxDIdSZxPhyw.png">(view full size)</a> |







Commit 619d4aea7d6727954c6d850b47f9275eae6c74eb (https://shuriken-mpevm5eip-cesiuminho.vercel.app).

File: pages/index.js [hunk 1]
```diff
@@ -1,31 +1,94 @@
...
-        <a href="/login" className={styles.button}>
-          Go to login
-        </a>
-      </main>
+const Home = () => (
+  <>
+    <Header />
+    <div className="xl:bg-yin-yang-curve-xl bg-cover bg-center -mb-10 md:mb-0">
+      <Hero />
+      <div className="bg-yin-yang-curve xl:bg-none bg-cover bg-center -mb-10 md:mb-0">
```
Comment (feliciofilipe) [inline pages/index.js:8]: I think we should align this  div margin-wise with the navbar information.

**Review by feliciofilipe — COMMENTED** [review]
Great Work :rocket:

**Comment by miguelbrandao** [comment]
Wow :open_mouth: The design looks great! :heart_eyes: 

The only major thing I see that needs to be improved is the background curve: it's too pixelated, it needs to be an SVG :stuck_out_tongue:

**Comment by nelsonmestevao** [comment]
> The only major thing I see that needs to be improved is the background curve: it's too pixelated, it needs to be an SVG stuck_out_tongue

Yes! For sure! It also needs some what different background for different break points. @feliciofilipe is going to ask Grabriela to work with him on this. Also, every image should become a SVG (the logo, the draw).

**Comment by miguelbrandao** [comment]
Another thing I just noticed: the site looks good in mobile and tablet mode, but it looks a bit weird in a 16:9 or wider aspect ratio (desktop).

You can see what I mean in (and my suggestions) in the image below:

image

I think we could improve on it by aligning the right and left content margins to the margins of the navbar (see :green_circle: lines). 
I also think the text portions (red boxes 1 and 3) are too wide in this screen size and that makes them a bit harder to read. Placing a max-width attribute in order for them too look a bit more like blue boxes (2 and 4) could be an improvement :smiley:

**Comment by nelsonmestevao** [comment]
> Another thing I just noticed: the site looks good in mobile and tablet mode, but it looks a bit weird in a 16:9 or wider aspect ratio (desktop).


Will do 💪

**Review by feliciofilipe — APPROVED** [review]
Amazing work :rocket: I would maybe increase the margin between the first text and the wave.

**Pull request merged**